aspnet/conceptual-overview/understanding-aspnet5-apps.rst

```diff
@@ -36,194 +27,93 @@ has been refactored into files that are generally smaller and more focused.
-
-Accessing configuration data from your app is best done by injecting the `IConfiguration <https://github.com/aspnet/Configuration/blob/dev/src/Microsoft.Framework.ConfigurationModel.Interfaces/IConfiguration.cs>`_ interface into your controller, and then simply using the indexer with the name of the configuration element you need. For example, to store the application name in config and display it on the About page, you would need to make three changes to the default project. First, add the entry to *appsettings.json*.
-
-.. code-block:: JSON
-	:emphasize-lines: 11
-
-	{
-		"Data": {
-			"DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet5-FundamentalConcepts"
-		},
-
-		"Entity": {
-			"ApplicationDbContext": {
-				"ConnectionString": "Data:DefaultConnection:ConnectionString"
-			}
-		},
-		"ApplicationName":"Fundamental Concepts"
-	}
-
-Next, make sure ASP.NET knows what to return when a constructor requires an instance of ``IConfiguration``. In this case, we can specify that the configuration value is a service. We'll address *Startup.cs* in a moment, but for this step just add one line to the end of the ``ConfigureServices()`` method in *Startup.cs*:
-
-.. code-block:: c#
-
-	services.AddInstance(typeof(IConfiguration), Configuration);
-
-The third and final step is to specify that your controller expects an ``IConfiguration`` instance via its constructor. Following the `Explicit Dependencies Principle`_ with your classes is a good habit to get into, and will allow ASP.NET 5's built-in support for Dependency Injection to work correctly. Assign the instance to a local field, and then access the configuration value by utilizing its' indexer on this instance.
-
-.. _`Explicit Dependencies Principle`: http://deviq.com/explicit-dependencies-principle/
-
-You will need to ensure you have this using statement:
-
-.. code-block:: c#
-
-	using Microsoft.Framework.Configuration;
-
-Then, update the controller as shown:
-
-.. code-block:: c#
-	:emphasize-lines: 3,5-7,9-15
-
-	public class HomeController : Controller
-	{
-		private readonly IConfiguration _config;
-
-		public HomeController(IConfiguration config){
-			_config = config;
-		}
-
-		public IActionResult About()
-		{
-			string appName = _config["ApplicationName"];
-			ViewData["Message"] = "You application name: " + appName;
-
-			return View();
-		}
-
-		public IActionResult Index(){
-			return View();
-		}
-	}
-
-
-Run the application and navigate to the About page and you should see the result.
-
-.. image:: understanding-aspnet5-apps/_static/about-page.png
+For more information, see `NuGet <https://docs.nuget.org>`_ and :doc:`/fundamentals/configuration`.
```

Comment:
The link to configuration really doesn't belong in this section. It should appear where you talk about configuration in the startup section below.